public/assets/css/app.css

```diff
@@ -215,6 +215,10 @@ table.break-words td, table.break-words th {
   word-wrap: break-word;
 }
 
+.ico_col {
```

Comment:
Sorry to be a pain but can we remove this and use the `text-center` class provided by [bootstrap](http://getbootstrap.com/css/) (this a dependency of ours already so we might as well use it). I should have mentioned this before sorry.